clang/test/CXX/drs/dr16xx.cpp

```diff
@@ -107,6 +107,7 @@ namespace dr1638 { // dr1638: 3.1
   struct B {
     friend enum class A<unsigned>::E;
     // since-cxx11-error@-1 {{reference to enumeration must use 'enum' not 'enum class'}}
+    // since-cxx11-error@-2 {{cannot be a friend}}
```

Comment:
Can you match the whole diagnostic message, like the rest of the tests in the file?